packages/frontend/app/pods/components/pod/chat/chat-entry/component.ts

```diff
@@ -79,7 +80,11 @@ export default class ChatEntry extends Component<IArgs> {
   @action onInput(event: KeyboardEvent) {
     const value = (event.target as any).value;
 
-    this.text = value;
+    if (value.charAt(value.length - 1) === ':') {
```

Comment:
what happens when you go back and add an emoji somewhere in the middle of your text?

Reply:
Fixed, thanks!